fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -609,6 +609,13 @@ public class Config extends ConfigBase {
     @ConfField
     public static int http_max_chunk_size = 8192;
 
+    /**
+     * If this time is exceeded when requesting the http service, a log will be output to remind you to determine
+     * the problem. Normally, the execution time of the metrics interface should not exceed 5 seconds.
```

Comment:
If a request takes longer than the configured time, a log will be generated to trace it.